src/components/FormBase.tsx

```diff
@@ -31,11 +33,16 @@ const FormBase: React.FC<Props> = (props) => {
           </Stack>
         </Stack>
 
-        <ButtonGroup variant="outline" spacing="4" width="full">
+        <ButtonGroup variant="outline" spacing="4" width="full" display={{base: "none", md: "flex" }}>
           <GoogleProvider />
           <GitHubProvider />
         </ButtonGroup>
 
+        <Stack spacing="4" width="full" display={{base: "flex", md: "none" }}>
+          <GoogleProvider variant="outline"/>
+          <GitHubProvider variant="outline"/>
+        </Stack>
+
```

Comment:
I feel like there is a better way to do this than by duplicating the providers twice? Could you make change the flex type based on the screen width instead of having it twice?